recipes/sdl2/bld.bat

```diff
@@ -0,0 +1,40 @@
+:: MSVC library for Python 2.7 is not found automaticaly if it is 
+:: installed in the user's AppData folder. Add it to PATH
+setlocal EnableDelayedExpansion
+
+set TARGET=Release
+:: Set target back to Debug for 64-bits builds with VS2008 and VS2010
+if %VS_MAJOR% LSS 14 (
+	if %ARCH% == 64 (
+		set TARGET=Debug
+	)
+)
+
+:: When installing Visual C++ compiler tools for Python 2.7, all files are placed in the users AppData folder
+:: conda-bld tries to call vcvarsall at its classic location, which of course fails, so we have to call it from its
+:: new location in the the users appdata folder manually, but only if this folder exists of course.
+if "%PY_VER%"=="2.7" (

```

Comment:
[OpenCV also fails](https://ci.appveyor.com/project/conda-forge/opencv-feedstock/build/1.0.63/job/7f0tnchs8ejbbttt#L286) to grab the `amd64` compiler due the configuration files being missing with the vanilla MSVC 2008. However, this doesn't matter as CMake still picks up the correct compiler for NMake files. Are you sure it doesn't work without all this logic?